src/compiler/scala/tools/nsc/typechecker/Implicits.scala

```diff
@@ -1205,7 +1215,7 @@ trait Implicits extends splain.SplainData {
                 val res = typedImplicit(firstPending, ptChecked = true, isLocalToCallsite)
                 if (res.isFailure) res
                 else
-                  new SearchResult(res.tree, res.subst, res.undetparams, firstPending)
+                  new SearchResult(res.tree, res.subst, res.undetparams, res.inPackagePrefix, implicitInfo = firstPending)
```

Comment:
@som-snytt this one was not obvious to catch 🎣 (https://github.com/scala/scala/commit/5532ca5021770b01fda2d6e33f6216387ed88004#diff-e67deea6ab1bc2d39cd27429daac7871803ec55f7d800103fe12503de15f5048R1211)

Reply:
I almost made SearchResult a case class, so this would be `copy`, to add `implicitInfo`.

Suddenly everyone wants to load up SearchResult with useful information.